src/components/BookingPage.js

```diff
@@ -0,0 +1,161 @@
+import { useContext, useEffect, useState } from "react";
+import { useLocation } from "react-router-dom";
+import screen from "./screen.svg";
+import whiteSeat from "./whiteSeat.svg";
+import greySeat from "./greySeat.svg";
+import blueSeat from "./blueSeat.svg";
+import {
+  AllSeatsStyle,
+  BookingPageStyle,
+  ConfirmModalStyle,
+  ModalCloseStyle,
+  RowSeatStyle,
+  ScreenStyle,
+  SeatNumberStyle,
+  SeatStyle,
+  SittingSeatStyle,
+} from "../styles/BookingPage.styled";
+import Context from "./context";
+import { GlobalStyle } from "../styles/GLobal.styled";
+import ConfirmBooking from "./ConfirmBooking";
+
+export default function BookingPage(props) {
+  const IMG_URL = "https://image.tmdb.org/t/p/w500";
+
+  const [selectedSeats, setSelectedSeats] = useState([]);
+  let [confirmedSeats, setConfirmedSeats] = useState([]);
+  const [modalVisible, setModalVisible] = useState(false);
+  const [bookingHomeVisible, setBookingHomeVisible] = useState(true);
+  let seatsForPriceCalc = [];
+  const data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+  const seats = [
+    {
+      row: "A",
+      seat: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
```

Comment:
You can make use of data array here in all the 4 cases instead of redeclaring it again and again.